src/Examples/DAE/petsc_pid_example.ipynb

```diff
@@ -0,0 +1,776 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.fs.tank.control_volume.properties_out[12].pressure)])\n",
+    "a = plt.ylabel(\"tank pressure (Pa)\")\n",
+    "a = plt.xlabel(\"time (s)\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0933779-f4b3-4933-83dc-0ec70645af45",
+   "metadata": {},
+   "source": [
+    "## Model a ramp in inlet perssure\n",
+    "\n",
+    "Next we show how to add an explicit time variable and ramp the inlet pressure from 500 kPa to 600 kPa between 10 and 12 seconds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "faba65e5-4b4d-46dc-ba74-10ff0424cd3d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "pressure : Pressure\n",
+      "    Size=1, Index=None, Units=Pa\n",
+      "    Key  : Lower : Value    : Upper        : Fixed : Stale : Domain\n",
+      "    None :   0.1 : 500000.0 : 1000000000.0 :  True :  True : PositiveReals\n"
+     ]
+    }
+   ],
+   "source": [
+    "# First, with the model we have, verify that the inlet pressure is fixed at 500 kPa.\n",
+    "m.fs.valve_1.control_volume.properties_in[0].pressure.display()"
```

Comment:
It's a little confusing that you do this check on the first model and then overwrite that model in the next cell. Why not wait and do this check on the extended-time model constructed below?

Reply:
I'm not totally sure what my point was with this.  I cut it.